.github/workflows/check_verilog.yml

```diff
@@ -0,0 +1,95 @@
+name: check-verilog
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+env:
+  CIRCT_VERSION: 1.72.0
+  CIRCT_ARCHIVE: "circt-full-shared-linux-x64.tar.gz"
+  DEPENDENCIES_DIR: "deps"
+  MLIR_TARGETS_PATH: "lib/cmake/mlir/MLIRTargets.cmake"
+  BUILD_DIR: "build"
+  UTOPIA_EXECUTABLE: "src/umain"
+  MODULE_OUTPUT_PATH: "output.sv"
+  LIBRARY_OUTPUT_PATH: "lib.sv"
```

Comment:
These environment variables are used at specific jobs/steps only. It would be better to declare them locally (well, as-local-as-possible).

Reply:
Fixed.